glommio/tests/linters.rs

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: © 2021 ChiselStrike <info@chiselstrike.com>
+
+#[cfg(test)]
+mod tests {
+    use std::process::Command;
+    #[test]
+    fn check_formating() {
+        let status = Command::new("cargo")
+            .args(["+nightly", "fmt", "--all", "--", "--check"])
+            .status()
+            .unwrap();
+        assert!(status.success());
```

Comment:
Can you add a failed assertion message here? A common mistake is to format using the stable channel so a little hint there would help people understand what's going on!